web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -133,6 +136,25 @@ public boolean include(Table table) {
         };
     }
 
+    @GET
+    @Path ("_changedtables")
+    @Timed (name = "bv.emodb.sor.DataStoreResource1.listMetadataChangedTables", absolute = true)
+    @Unbuffered
+    @ApiOperation (value = "Returns all the emo tables whose metadata was changed.",
+            notes = "Returns a Iterator of a Map.",
+            response = Table.class
+    )
+    public Iterator<Map<String, Object>> listMetadataChangedTables(final @QueryParam("from") DateTimeParam fromParam,
```

Comment:
Should put a new permission on this, or at least the same permission as listing tables.  If the latter this should filter out tables for which the user doesn't have "read" access.